src/plugins/plugin.legend.js

```diff
@@ -528,15 +528,60 @@ class Legend extends Element {
 		const position = titleOpts.position;
 		let x, textAlign;
 
-		if (position === 'start') {
-			x = me.left + titlePadding.left;
+		const halfFontSize = titleFont.size / 2;
+		let y = me.top + titlePadding.top + halfFontSize;
+
+		// These defaults are used when the legend is vertical.
+		// When horizontal, they are computed below.
+		let left = me.left;
+		let maxWidth = me.width;
+
+		if (this.isHorizontal()) {
+			// Move left / right so that the title is above the legend lines
+			maxWidth = Math.max(...me.lineWidths);
+			switch (opts.align) {
+			case 'start':
+				// x is already correct in this case
```

Comment:
this comment should maybe be about `left` here, and the assignment below can be omitted

Reply:
You're right, I've updated the comment and removed the extra assignment.